ocs_ci/utility/lvmo_utils.py

```diff
@@ -56,3 +63,68 @@ def lvmo_health_check(tries=20, delay=30):
         delay=delay,
         backoff=1,
     )(lvmo_health_check_base)()
+
+
+def get_sno_disks_by_path():
```

Comment:
I would suggest passing the node name as a argument and use constant as a default value to it.

Reply:
added + docstring